Gruntfile.coffee

```diff
@@ -1324,7 +1337,27 @@ module.exports = (grunt) ->
 					base: "<%= coreDist %>"
 					message: ((
 						if process.env.TRAVIS_TAG
-							"Variant files for the " + process.env.TRAVIS_TAG + " maintenance release"
```

Comment:
@pjackson28 can we merge as is and perhaps revist the message in another issues since it spans multiple repos?